symfony/plugins/orangehrmAdminPlugin/Dao/CountryDao.php

```diff
@@ -17,26 +15,30 @@
  * You should have received a copy of the GNU General Public License along with this program;
  * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
  * Boston, MA  02110-1301, USA
- * 
  */
 
-class CountryDao extends BaseDao {
+namespace OrangeHRM\Admin\Dao;
+
+use OrangeHRM\Entity\Country;
+use OrangeHRM\Core\Dao\BaseDao;
+use DaoException;
+use Exception;
 
+class CountryDao extends BaseDao
+{
     /**
      * Get Country list
-     * @return Country
+     * @return Country[]
+     * @throws DaoException
      */
-    public function getCountryList() {
+    public function getCountryList(): array
+    {
         try {
-            $q = Doctrine_Query::create()
-                    ->from('Country c')
-                    ->orderBy('c.name');
-
-            $countryList = $q->execute();
-
-            return $countryList;
+            return $this->getRepository(
+                Country::class
+            )->findAll();
```

Comment:
ordering missing from the old implementation. better to use `$this->createQueryBuilder`